annotations/kubernetes-annotations/src/main/java/io/dekorate/kubernetes/listener/KubernetesSessionListener.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2018 The original authors.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+**/
+
+package io.dekorate.kubernetes.listener;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import io.dekorate.BuildService;
+import io.dekorate.BuildServiceFactories;
+import io.dekorate.DekorateException;
+import io.dekorate.Resources;
+import io.dekorate.Session;
+import io.dekorate.SessionListener;
+import io.dekorate.WithProject;
+import io.dekorate.WithSession;
+import io.dekorate.deps.kubernetes.api.model.KubernetesList;
+import io.dekorate.deps.kubernetes.client.DefaultKubernetesClient;
+import io.dekorate.deps.kubernetes.client.KubernetesClient;
+import io.dekorate.hook.ImageBuildHook;
+import io.dekorate.hook.ImagePushHook;
+import io.dekorate.hook.OrderedHook;
+import io.dekorate.hook.ProjectHook;
+import io.dekorate.kubernetes.config.ImageConfiguration;
+import io.dekorate.kubernetes.config.KubernetesConfig;
+import io.dekorate.kubernetes.hook.ScaleDeploymentHook;
+import io.dekorate.project.Project;
+
+public class KubernetesSessionListener implements SessionListener, WithProject, WithSession {
+
+  private static final String KUBERNETES = "kubernetes";
+
+	@Override
+	public void onClosed() {
+    Session session = getSession();
```

Comment:
The formatting looks a little weird here

Reply:
It's two spaces in, if you check the file.
However, in your comment it does look like there is no space. Weird!

Reply:
Weird indeed but not something to worry about :)